deployments/_deployments_log_file.json

```diff
@@ -669,6 +669,20 @@
           }
         ]
       }
+    },
+    "kaia": {
+      "production": {
+        "1.0.0": [
+          {
+            "ADDRESS": "0x8176C5502f1A01fCeB8d14fdc7ad15867ca562C1",
+            "OPTIMIZER_RUNS": "1000000",
+            "TIMESTAMP": "2024-09-05 16:24:10",
+            "CONSTRUCTOR_ARGS": "0x",
+            "SALT": "",
+            "VERIFIED": "true"
```

Comment:
https://kaiascan.io/address/0x8176c5502f1a01fceb8d14fdc7ad15867ca562c1?tabId=contract&page=1

This does not seem verified to me

Reply:
This was the official explorer as far as I was aware https://kaiascope.com/account/0x8176C5502f1A01fCeB8d14fdc7ad15867ca562C1?tabId=internalTx

Reply:
May be....I found this other one linked on the homepage of Kaia chain (bottom right):
https://www.kaia.io/

But OK, as long as we have them verified in one explorer we can point to that...just thought the verification didnt work at all